recipes/ray/meta.yaml

```diff
@@ -0,0 +1,70 @@
+{% set version = "0.8.0" %}
+
+package:
+  name: ray
```

Comment:
Please change the package name to `prefix-ray`/`ray-postfix` since we already have a `bioconda::ray` package: https://github.com/bioconda/bioconda-recipes/blob/fa79535e46d0e4bfb542516d9eef22f3e183e6c7/recipes/ray/meta.yaml
(Bioconda is a project we collaborate with.)

`prefix`/`postfix` can either be something related to the ecosystem the package originates from, e.g., for PyPI `python-` is a common prefix, or it can be something related to the nature of the software.

Reply:
How are other packages in conflict with bioconda dealing with this? I think the package becomes very hard to discover if we diverge from the name on PyPI.